src/prefect/engine/result/base.py

```diff
@@ -80,12 +80,6 @@ class Result(ResultInterface):
         - validators (Iterable[Callable], optional): Iterable of validation functions to apply to
             the result to ensure it is `valid`.
         - run_validators (bool): Whether the result value should be validated.
-        - cache_for (timedelta, optional): The amount of time to maintain a cache
-            of this result.  Useful for situations where the containing Flow
-            will be rerun multiple times, but this task doesn't need to be.
-        - cache_validator (Callable, optional): Validator that will determine
-            whether the cache for this result is still valid (only required if `cache_for`
-            is provided; defaults to `prefect.engine.cache_validators.duration_only`)
```

Comment:
Why removed and not deprecated?

Reply:
They never did anything so no need to deprecate

Reply:
Oh oh I thought this was the task.py file haha